test/commands/submission/analysis/init_test.rb

```diff
@@ -22,7 +22,36 @@ class Submission::Analysis::InitTest < ActiveSupport::TestCase
         exercise_git_repo: "ruby",
         exercise_git_sha: solution.git_sha,
         exercise_git_dir: exercise_repo.dir,
-        exercise_filepaths: [".meta/config.json", ".meta/design.md", ".meta/exemplar.rb"]
+        exercise_filepaths: [".meta/config.json", ".meta/design.md", ".meta/exemplar.rb", "log_line_parser_test.rb"]
+      }
+    )
+    Submission::Analysis::Init.(submission)
+    assert_equal 'queued', submission.analysis_status
+  end
+
+  test "calls to publish_message for exercise with editor files" do
```

Comment:
This test case verifies that the original issue (editor files not being present)